openshift-gitops-operator/tasks/main.yml

```diff
@@ -0,0 +1,191 @@
+---
+- name: Setting Up Results Var
+  ansible.builtin.set_fact:
+    test_result: "pass"
+- name: Test ArgoCD with Application
+  block:
+    # 1. Create Test Project
+    - name: Create test gitops namespace
+      kubernetes.core.k8s:
+        name: test-gitops
+        api_version: v1
+        kind: Namespace
+        state: present
+    # 2. Testing argocd Instance
+    - name: Create argocd
+      kubernetes.core.k8s:
+        name: argocd
+        namespace: test-gitops
+        api_version: argoproj.io/v1alpha1
+        kind: ArgoCD
+        definition:
+          metadata:
```

Comment:
Given the size of this, I'd put it in a template.

Reply:
Done.